skorch/net.py

```diff
@@ -1325,6 +1327,44 @@ def load_params(self, f):
 
         self.module_.load_state_dict(model)
 
+    def save_history(self, f):
+        """Saves the history of ``NeuralNet`` as a json file.
+
+        Parameters
+        ----------
+        f : file-like object or str
+
+        Examples
+        --------
+        >>> before = NeuralNetClassifier(mymodule)
+        >>> before.save_history('path/to/file.json')
+        >>> before.fit(X, y)
+        >>> after = NeuralNetClassifier(mymodule).initialize()
+        >>> after.load_history('path/to/file.json')
```

Comment:
This example is broken or at least misleading: if `mymodule` is a class then `after` is a freshly initialized net/module with the history of a trained one which will surely result in unexpected behavior. I think it would be better to show this functionality with `net.load`/`net.save`.